packages/back-end/src/controllers/features.ts

```diff
@@ -166,6 +172,7 @@ export async function getPayloadParamsFromApiKey(
 
     return {
       organization,
+      capabilities: [],
```

Comment:
Is this going to break payloads for legacy API keys?  I'm pretty sure those payloads include hashing v2 fields today.

Reply:
updated the scrubbing to be multi-staged (and passed in capabilities here) based on our convo...